main.tf

```
@@ -0,0 +1,279 @@
+locals {
+  enabled                      = module.this.enabled
+  network_firewall_name        = var.network_firewall_name != null && var.network_firewall_name != "" ? var.network_firewall_name : module.this.id
```

There's a duplicate here
```suggestion
  network_firewall_name        = var.network_firewall_name != null ? var.network_firewall_name : module.this.id
```